website/content/docs/site-generator-overview.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Overview
+group: Guides
+weight: 1
+---
+
+A common pain point with static site generators is that they are hard to use for non-technical editors. Content and source code are stored together in the same Git repository. Making changes to the site content requires editors to go through a sequence of steps:
+
+1. Navigating to the site repository
+2. Creating or editing content files such as Markdown, JSON, YAML or TOML files
+3. Staging and committing changes to the local repository
+4. Pushing changes to the remote repository, which will trigger a site build and deployment
+
+Netlify CMS fills this gap. You can integrate it into any site built with a common static site generator, including but not limited to Gatsby, Hugo, Jekyll, Next.js, Nuxt.js, Middleman and Gridsome. This allows non-technical editors to create and edit content through a solid user interface, abstracting any Git related operations.
+
+The process for adding Netlify CMS to your site can vary between different generators. As a general approach, it has three main steps:
+
+1. Adding Netlify CMS to your site under the `/admin` route of the published site
+2. Setting up a backend to authenticate CMS users with the Git provider that hosts your source code
+3. Rendering the Netlify CMS generated content as web pages
+
+## Adding Netlify CMS to your site
+
+The frontend of the app contains the following major components:
+
+- A dashboard for managing content
+- A rich text editor built with [Slate](https://github.com/ianstormtaylor/slate) that supports Markdown syntax
+- A preview pane that renders the content while it's being edited
+
+Some site generators (e.g. Gatsby) come with plugins that simplify the installation of the Netlify CMS app.
+
+## Authenticate CMS users with a backend
+
+Netlify CMS doesn't store your site content. Think of it as a middleman that allows your CMS users to make changes to your Git repository without having an account with GitHub, GitLab etc.
+
+To achieve that, Netlify CMS needs to make sure that users have the write access to an online Git repository. This means the backend of the app is just JavaScript code that authenticates users with the Git provider which stores the site project.
+
+The authentication process requires a server and setting up a backend. You can write your own backend, or deploy your site to Netlify and get two authentication micro-services that simplify this process: Netlify Identity and Git Gateway.
+
+It is also helpful to remember that Netlify CMS works in production. Every time the repository changes, whether in the source codes or raw content, the website is rebuilt and redeployed automatically.
+
+For experimentation, you can run Netlify CMS locally, albeit with a catch. Each content edit through Netlify CMS results in a new commit to your remote repository. This means that you will need to pull changes from your remote each time to see them in the locally served site.
```

Comment:
Oh, it's a new feature? I remember doing several mental gymnastics when trying to make Netlify CMS app work with a local repo a few months ago. Should I delete both paragraphs and link to that beta feature?

Reply:
> Oh, it's a new feature?

A few months old I believe. Probably less visible since it's a beta feature.

> Should I delete both paragraphs and link to that beta feature?

Yes, that would be great!